bool state
---
bool status
string message
